### ISSUE TITLE
Document the process of auditing and migrating external GitHub repositories to pulibrary

### ISSUE THREAD

**Issue by jrgriffiniii** [pulibrary/pul-the-hard-way#62]
There was at one case which arose for this, where I wanted to migrate https://github.com/pulibrary/dspace-development from my personal GitHub account. While I was provided with ample assistance and guidance, I would be curious as to whether or not there should be guidelines around quality assurance for these types of cases (one cannot move a repository unless there is an explicit license, code of conduct, README, testing and code coverage, etc.)

**Comment by christinach** [comment]
I agree @jrgriffiniii  it would be very useful to have this documented.
